include/mbedtls/md.h

```diff
@@ -99,7 +105,7 @@ typedef struct mbedtls_md_context_t
     const mbedtls_md_info_t *md_info;
 
     /** The digest-specific context. */
-    void *md_ctx;
+    void *MBEDTLS_PRIVATE(md_ctx);
```

Comment:
As is, this causes Doxygen documentation to list an entry for
> void* mbedtls_md_context_t::MBEDTLS_PRIVATE ( md_ctx  )

I'm not sure if that's good or bad.

Reply:
I think in the long run we should probably tune doxygen so that it stops listing private fields when generating public API documentation, which would made the question go away. In the meantime, I think the current behaviour is OK.